applications/GeoMechanicsApplication/tests/test_parameter_field.py

```diff
@@ -65,9 +66,27 @@ def test_parameter_field_with_python_umat_parameters(self):
 
         # copy user defined python script to installation folder
         new_custom_script_path = os.path.join(os.path.dirname(KratosGeo.__file__), "user_defined_scripts")
-        shutil.copy(custom_python_file,new_custom_script_path)
-
-        # run simulation
+        try:
+            shutil.copy(custom_python_file, new_custom_script_path)
+        except shutil.SameFileError as e:
+             print('Source and destination represents the same file. %s' % e)
+             raise
+        except PermissionError as e:
+             print('Permission denied: %s' % e)
+             print('The file %s' % new_custom_script_path)
+             raise
+        except IOError as e:
+             print('Try to change permission %s' % e)
+             print('The file %s' % new_custom_script_path)
+             os.chmod(new_custom_script_path, stat.S_IRWXU)
+             shutil.copy(custom_python_file, new_custom_script_path)
+        except:
```

Comment:
This catches any other kind of exception, which is a good thing to do. I would, however, prefer to print a bit more information for this case by catching any `Exception` instance, which you can then refer to as `e`:
```suggestion
        except Exception as e:
```

Subsequently, you can use `e` to make the error message more descriptive (like you did for the other kinds of errors).

You may want to apply this suggestion also to the other `try: {...} except: {...}` block that you've modified.

Reply:
added in both places.